src/com/javacore/lesson09/User.java

```diff
@@ -0,0 +1,119 @@
+package com.javacore.lesson09;
+
+public class User {
+
+    String userName;
+    String userEmail;
+    String userPhoneNumber;
+    String userPassword;
+
+    private User() {
+
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    private boolean verificator(String parameter, String regex) throws InvalidInputException {
```

Comment:
if method returns boolean, name should display this
isValid for example
or as usual methods, use verb - validate
you cannot name methods as nouns, it's not an object it's action